src/js/webflow/import.js

```diff
@@ -15,7 +15,7 @@ exports.transform = function (window, $) {
   $("head").append(`
     <script>
       const pushpadProjectId = \`${process.env.PUSHPAD_PROJECT_ID}\`;
-      if (pushpadProjectId !== '') {
+      if (pushpadProjectId !== '' && pushpadProjectId !== 'undefined') {
```

Comment:
I believe it should be as in the suggestion further below?

In the current form, the following code will print the console statement:

```
let pushpadProjectId;

function test() {
  if (pushpadProjectId !== '' && pushpadProjectId !=='undefined') {
    console.log("Hello fron inside the if");
  }
}
```

With the change below, it will not go into the `if` statement which is what we want.

```suggestion
      if (pushpadProjectId !== '' && typeof pushpadProjectId !== 'undefined') {
```

Reply:
We are initialising it inside a template literal, so if the env var is `undefined` it ends up being a string `'undefined'`:
`const pushpadProjectId = \`${process.env.PUSHPAD_PROJECT_ID}\``

Reply:
I see, we are checking for the literal string. I would add a code comment to this to make it clear for our future selves 😃

Reply:
I added a comment to explain it 👍🏼